src/reducers/EndpointSearchReducers.js

```diff
@@ -78,20 +64,6 @@ export const friendTextLocation = (state = '', action) => {
   }
 };
 
-// export const friendGPSLocation = (state = {lat: 0, lng: 0}, action) => {
```

Comment:
Remove all the dead code. Great workflow and atomic commits!

![](https://18670-presscdn-pagely.netdna-ssl.com/wp-content/uploads/2016/08/questionable-hand-movements-5.gif)